CONTRIBUTING.md

```diff
@@ -76,3 +76,61 @@ You should, of course, adapt this header to use the specific mechanism for comme
+1. Ensure your Chart changes follow the [technical](#technical-requirements) and [documentation](#documentation-requirements) guidelines, described below.
+1. Submit a pull request.
+
+***NOTE***: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+
+### Technical Requirements
+
+* All Chart dependencies should also be submitted independently
+* Must pass the linter (`helm lint`)
+* Must successfully launch with default values (`helm install .`)
+    * All pods go to the running state (or NOTES.txt provides further instructions if a required value is missing e.g. [minecraft](https://github.com/helm/charts/blob/master/stable/minecraft/templates/NOTES.txt#L3))
+    * All services have at least one endpoint
+* Must include source GitHub repositories for images used in the Chart
+* Images should not have any major security vulnerabilities
+* Must be up-to-date with the latest stable Helm/Kubernetes features
+    * Use Deployments in favor of ReplicationControllers
+* Should follow Kubernetes best practices
+    * Include Health Checks wherever practical
+    * Allow configurable [resource requests and limits](http://kubernetes.io/docs/user-guide/compute-resources/#resource-requests-and-limits-of-pod-and-container)
+* Provide a method for data persistence (if applicable)
+* Support application upgrades
+* Allow customization of the application configuration
+* Provide a secure default configuration
+* Do not leverage alpha features of Kubernetes
+* Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
+* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
+  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
+  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+
+### Documentation Requirements
+
+* Must include an in-depth `README.md`, including:
+    * Short description of the Chart
+    * Any prerequisites or requirements
+    * Customization: explaining options in `values.yaml` and their defaults
+* Must include a short `NOTES.txt`, including:
+    * Any relevant post-installation information for the Chart
+    * Instructions on how to access the application or service provided by the Chart
+
+### Merge Approval and Release Process
+
+All pull requests will be verified by a CI pipeline which runs the linter an tries to install the chart to at least the three most recent minor releases of kubernetes.
+IA maintainer may add "LGTM" (Looks Good To Me) or an equivalent comment to indicate that a PR is acceptable. Any change requires at least one LGTM. No pull requests can be merged until at least one maintainer signs off with an LGTM.
```

Comment:
This has its origin by the [k8s prow bot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home), which actually merge PRs if a maintainer comment with LGTM.

Since we are not using this bot, I would adjust this point that a maintainer must agree somehow.

Reply:
I'll adapt the wording accordingly